### PR TITLE
Monitoring config changes

### PR DESCRIPTION
**Pull request by qnguyen3411**
Created sections for config file. Moved log file & host file path variables to config file. Fixed monitoring-client and grafana so that they use these path variables. Fixed README to match these changes.

### CODE REVIEW

**Review by the-auk — APPROVED** [review]

File: monitoring/grafana.sh [hunk 1]
```diff
@@ -1,17 +1,8 @@
 #!/bin/bash
 
-cd --
+# cd --
```
Comment (breed) [inline monitoring/grafana.sh]: please just delete this line

File: monitoring/grafana.sh [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
 python3 -m pip install bottle
```
Comment (breed) [inline monitoring/grafana.sh]: hmmm this whole script is pretty sketchy...

File: monitoring/grafana.py [hunk 3]
```diff
@@ -2,10 +2,15 @@
 from sys import argv
 from datetime import datetime
 from calendar import timegm
+import configparser
 from bottle import (Bottle, HTTPResponse, run, request, response,
                     json_dumps as dumps)
+
+monitor_config = configparser.ConfigParser()
+monitor_config.read('monitoring.config')
```
Comment (breed) [inline monitoring/grafana.py]: can you pass the path on the command line. reading from the current working directory can be problematic.

File: monitoring/monitoring_piman.sh [hunk 4]
```diff
@@ -1,22 +1,12 @@
 #!/bin/bash
 
 # set up ip table for the pi
-cd --
+# cd --
```
Comment (breed) [inline monitoring/monitoring_piman.sh]: remove

**Review by breed — COMMENTED** [review]
looks pretty good. just a couple of requested changes.

File: monitoring/grafana.py (hunk 3 above)
Comment (qnguyen3411) [reply monitoring/grafana.py]: The main piman config parser uses glob to resolve its config file, should we try that or should we stick with passing in the path to config as a command line argument?

**Pull request closed without merging**